observatorium/overlays/moc/smaug/thanos/deployments/observatorium-thanos-query-frontend.yaml

```diff
@@ -0,0 +1,104 @@
+kind: Deployment
```

Comment:
seems like this already exists
https://github.com/operate-first/apps/blob/master/observatorium/base/instance/thanos/query-frontend/deployment.yaml